gateway/appconfig/appconfig.go

```diff
@@ -242,6 +243,9 @@ func hasIdpEnvs() bool {
 
 func loadPostgresCredentials() (*pgCredentials, error) {
 	pgConnectionURI := os.Getenv("POSTGRES_DB_URI")
+	if pgConnectionURI == "" {
+		pgConnectionURI = "postgres://postgres:postgres@localhost:5432/hoop?sslmode=disable"
+	}
```

Comment:
Since the goal of this PR is having an environment for development, I recommend using a check to validate if it's a development build. This will only add this default for these environments.

> Having a default value for production may lead to unwanted behaviors that are hard to troubleshoot on proof of concepts or production environments.

Example:

```go
import "github.com/hoophq/hoop/common/version"
(...)

isDevBuild := version.Get().Version == "unknown"
if isDevBuild && pgConnectionURI == "" {
  pgConnectionURI = "postgres://postgres:postgres@localhost:5432/hoop?sslmode=disable"
}